web/src/Routes.tsx

```diff
@@ -17,31 +17,19 @@
 import React, {lazy, Suspense} from 'react';
 
 import Loading from 'components/common/Loading';
-import CustomerCommitCountProvider from 'components/customer/contexts/CommitCountContext';
+import CustomerRoutes from 'components/customer/Routes';
+import MerchantRoutes from 'components/merchant/Routes';
 import {Switch, Route} from 'react-router-dom';
 
-// Customer Pages
-const CustomerExplorePage = lazy(() => import('components/customer/explore'));
-// Merchant Pages
-const MerchantLandingPage = lazy(() => import('components/merchant/landing'));
-const MerchantSignUpPage = lazy(() => import('components/merchant/sign-up'));
-const MerchantSignInPage = lazy(() => import('components/merchant/sign-in'));
-// Common Pages
-const ListingDetailsPage = lazy(() => import('components/listing-details'));
```

Comment:
Should this be common route?

Reply:
@ifhuang Hmm I don't think it makes much of a difference but since we are talking about the components here I think 'page' is more accurate.

Reply:
I think Yifu was asking about the ListingDetailsPage: Is this a common route shared between merchant and customer? If so, it should be moved into the customer routes alone; should have a separate Common Routes somewhere.

Reply:
@ifhuang @xlai20 @hzxcaryn To summarise what we discussed during today's meeting on this issue, previously listing route is common for both customer and merchant. If a customer/merchant goes to this route, they would be redirected to the customer/merchant's listing page respectively. However, for simplicity, we have decided not to have this route. There are now 2 separate routes for customer (/listing/:id) and merchant (/merchant/listing/:id).